components/script/dom/htmltextareaelement.rs

```diff
@@ -124,6 +129,8 @@ impl HTMLTextAreaElement {
             textinput: DomRefCell::new(TextInput::new(
                     Lines::Multiple, DOMString::new(), chan, None, None, SelectionDirection::None)),
             value_dirty: Cell::new(false),
+            maxlength: Cell::new(DEFAULT_MAX_LENGTH),
```

Comment:
I think these should be called `max_length` and `min_length` for consistency for the rest of the code.

Reply:
But these are actually unused, aren't they?